### DISCUSSION TITLE
TPI first day of heating

### DISCUSSION THREAD

**Discussion by prigal** [jmcollin78/versatile_thermostat#636] [Q&A] [answered]
Hello, 

Thanks for your work, I'm a user since last year. 

![Screenshot_20241114-212726](https://github.com/user-attachments/assets/022c8567-fedc-426e-a8c5-07d8aa4af5e0)

I did all updates and this is my first day of heating in my daughter's room. Quite old house not really well isolated.

Climate over switch (quite old oil bath radiator, and [Legrand cable outlet](https://github.com/Koenkk/zigbee2mqtt/issues/url)) Target 17°, TPI with default preset (0.6, 0.01)

House empty, came back at home at 19.

Does it looks ok ? I have the feeling that my heaters are starting and stopping way too often and that the target temperature is not really reached.

Thank you !

**Comment by jmcollin78** [comment, marked as answer]
Hello @prigal ,

This is encouraging for a first try but we should do much better. Because you seems to be French, you will be able to read this article which explains how to tune the VTherm for optimale performance. The article is here: https://www.hacf.fr/optimisation-versatile-thermostat/

Yours is really too slow. I let you read the article, you case is exactly describe on the article.

**Reply by prigal** [reply]
Thanks !

Merci pour l'article, très instructif, je vais essayer de régler tout ça !
